src/Raven.Client/Documents/Operations/Indexes/DisableIndexOperation.cs

```diff
@@ -2,42 +2,67 @@
 using System.Net.Http;
 using Raven.Client.Documents.Conventions;
 using Raven.Client.Http;
+using Raven.Client.Json;
 using Sparrow.Json;
 
 namespace Raven.Client.Documents.Operations.Indexes
 {
     public class DisableIndexOperation : IMaintenanceOperation
     {
-        private readonly string _indexName;
+        private readonly Parameters _parameters;
 
-        public DisableIndexOperation(string indexName)
+        public DisableIndexOperation(string indexName, bool clusterWide = false)
```

Comment:
Let's maintain binary compatibility, and create a separate overload.

Reply:
Also, where is the Enable op?

Reply:
Maybe create a new operation?
ToggleIndexState?

Reply:
No